### PR TITLE
VTCCA-3728: Fix IV failures after removing '@' from app.routes

### PR DESCRIPTION
**Pull request by darrell-plant**

### CODE REVIEW

**Comment by habeeb01** [comment]
LGTM 👍

File: conf/application.conf [hunk 1]
```diff
@@ -56,8 +56,8 @@ featureFlags {
   editNameEnabled = true
   fileUploadEnabled = false
   dvrEnabled = true
-  dvrCheckEnabled = false
-  clientPropertiesEnabled = false
+  dvrCheckEnabled = true
```
Comment (PeterPerhac) [inline conf/application.conf:59]: was this meant to be checked in?

**Comment by PeterPerhac** [comment]
👍  looks good, just wasn't sure if the toggling of flags was intentional in this PR

File: conf/application.conf (hunk 1 above)
Comment (darrell-plant) [reply conf/application.conf:59]: Yeah.

**Pull request merged**